cli/main.rs

```diff
@@ -412,6 +412,17 @@ async fn bundle_command(
   debug!(">>>>> bundle START");
   let global_state = GlobalState::new(flags)?;
 
+  global_state
+    .prepare_module_load(
+      module_specifier.clone(),
+      None,
+      TargetLib::Main,
+      Permissions::allow_all(),
+      false,
+      global_state.maybe_import_map.clone(),
+    )
+    .await?;
```

Comment:
@bartlomieju Is this ok?

Reply:
Not really - it will duplicate a lot of work. It looks like lock file abstraction is not yet fully complete, but I think we should just duplicate a bit of code for `bundle` path. @TETRA2000 please copy-paste these two bits into `ts_compiler.bundle()`:

```rust
let module_graph_files = module_graph.values().collect::<Vec<_>>();
    // Check integrity of every file in module graph
    if let Some(ref lockfile) = self.lockfile {
      let mut g = lockfile.lock().unwrap();

      for graph_file in &module_graph_files {
        let check_passed =
          g.check_or_insert(&graph_file.url, &graph_file.source_code);

        if !check_passed {
          eprintln!(
            "Subresource integrity check failed --lock={}\n{}",
            g.filename, graph_file.url
          );
          std::process::exit(10);
        }
      }
    }
```

and 

```rust
if let Some(ref lockfile) = self.lockfile {
      let g = lockfile.lock().unwrap();
      g.write()?;
    }
```

Reply:
@bartlomieju @ry
Thank you very much for your review!!
I fixed this PR according the comment.

Please take a look again.